cpp/ql/src/Likely Bugs/Memory Management/UsingExpiredStackAddress.cpp

```diff
@@ -0,0 +1,25 @@
+static const int* xptr;
+
+void localAddressEscapes() {
+  int x = 0;
+  xptr = &x;
+}
+
+void example1() {
+  localAddressEscapes();
+  const int* x = xptr; // BAD: This pointer points to expired stack allocated memory.
+}
+
+void localAddressDoesNotEscape() {
+  int x = 0;
+  xptr = &x;
+  // ...
+  // use `xptr`
+  // ...
+  xptr = nullptr;
+}
+
+void example2() {
+  localAddressEscapes();
```

Comment:
I think this should be:
```suggestion
  localAddressDoesNotEscape();
```
otherwise this example is identical to `example1`.

Reply:
Oh, right. Thanks a lot!